scripts/ci-checks.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Shell scripts
```

Comment:
Can we replace these comments with echos? Would be helpful to explicitly list the stages we're running, to help distinguish what has failed/needs fixing.